### PR TITLE
[docs] improve DebOps release schedule

### PR DESCRIPTION
**Pull request by nqb**

### CODE REVIEW

**Comment by nqb** [comment]
Some feedback about this documentation (as requested on IRC):
- Title: "DebOps release schedule" doesn't sound good for me. If I am a new DebOps user, I will search a document wit title like "Which DebOps version should I need to use ?"
- It would be nice to mention in this document which distributions are supported by DebOps
- We could simplify the section "How to use the rolling release ?" by making a link to a refreshed section about `debops-update` usage (like https://docs.debops.org/en/master/debops-tools/scripts/debops-update.html).
- For users using the "rolling" release, it will be nice to have a mailing-list use to send diff on `Changelog` file in order to track changes (in place of having to read it on a regular basis)
- To avoid confusion, I think we should use a special tag for the master branch. Currently on the `master` branch:
```console
$ git describe
v1.0.0-476-g8a0e866e9
```
- Same for unreleased stable in https://docs.debops.org/en/stable-1.0/news/changelog.html, I will prefer to see : `debops stable-1.X - unreleased` in place of `debops stable-1.0 - unreleased`

I will go in vacation for one week.

File: docs/news/releases.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ DebOps project is developed in a :command:`git` repository, with the ``master``
 branch as the main development branch. The project's repository is `hosted on
 GitHub`__, with a `mirror on GitLab`__ used for testing the Ansible roles via
 a GitLab CI pipeline. This release is meant for those that prefer to get the
-latest updates in the codebase, bugfixes and improvelements.
+latest updates in the codebase, bugfixes and improvements.
```
Comment (drybjed) [inline docs/news/releases.rst:22]: What the hell, how could I write that one? Thanks! :-)

**Review by drybjed — APPROVED** [review]
Thanks for the fixes! I'll merge that shortly, for the rest of the questions - I guess we can still talk in a closed PR.

**Comment by drybjed** [comment]
> * Title: "DebOps release schedule" doesn't sound good for me. If I am a new DebOps user, I will search a document wit title like "Which DebOps version should I need to use ?"

I want to avoid posing questions as documentation page titles. Also, the change you propose is very long, it would probably look awkward in the sidebar on https://docs.debops.org/. Any alternatives?

> * It would be nice to mention in this document which distributions are supported by DebOps

Sure, good idea. A short sentence with a link to the [longer explanation about software sources](https://docs.debops.org/en/master/dep/dep-0003.html) might be good, I'll try to work that in.

> * We could simplify the section "How to use the rolling release ?" by making a link to a refreshed section about `debops-update` usage (like https://docs.debops.org/en/master/debops-tools/scripts/debops-update.html).

Sounds good.

> * For users using the "rolling" release, it will be nice to have a mailing-list use to send diff on `Changelog` file in order to track changes (in place of having to read it on a regular basis)

I'm not sure I understand. Do you want to see regular updates on the mailing list about changes on the `master` branch? Can you suggest a schedule for that? :-)

> * To avoid confusion, I think we should use a special tag for the master branch. Currently on the `master` branch:
> 
> ```
> $ git describe
> v1.0.0-476-g8a0e866e9
> ```
I was considering that idea (you can see this in previous revisions in the `releases.rst` file), but other users thought that having multiple tags for `master` and `stable-x.y` branches might be too confusing. Perhaps if you explain how the "special tag" would work on the `master` branch (schedule-based? time-based? every 30 commits?) something could be designed.

> * Same for unreleased stable in https://docs.debops.org/en/stable-1.0/news/changelog.html, I will prefer to see : `debops stable-1.X - unreleased` in place of `debops stable-1.0 - unreleased`

The` stable-1.0` branch will stay `stable-1.0`, next branch will be created from the `master` branch and will be `stable-1.1`. So there's no direct way to move from the first one to the second one.

> I will go in vacation for one week.

Have a good time! I'm aiming for the `v1.1.0` release near the end of August, so don't worry, you still will have some time for input. :)

**Comment by nqb** [comment]
> I want to avoid posing questions as documentation page titles. Also, the change you propose is very long, it would probably look awkward in the sidebar on https://docs.debops.org/. Any alternatives?

I see you have replaced title by "DebOps releases" which sounds good for me. Anyway, we should add an entry in the FAQ. 

> Sure, good idea. A short sentence with a link to the [longer explanation about software sources](https://docs.debops.org/en/master/dep/dep-0003.html) might be good, I'll try to work that in.

In the DEP, I didn't see any references to Devuan and Raspbian, is it intended ?

> Do you want to see regular updates on the mailing list about changes on the `master` branch? 

Yes, but only for the Changelog file, each time this file is updated. It could simply be a diff.

**Comment by drybjed** [comment]
> > I want to avoid posing questions as documentation page titles. Also, the change you propose is very long, it would probably look awkward in the sidebar on https://docs.debops.org/. Any alternatives?
> 
> I see you have replaced title by "DebOps releases" which sounds good for me. Anyway, we should add an entry in the FAQ.

A FAQ entry about which version of DebOps to use? I don't think that's a frquently asked question yet, let's see if it becomes one. :-)

> > Sure, good idea. A short sentence with a link to the [longer explanation about software sources](https://docs.debops.org/en/master/dep/dep-0003.html) might be good, I'll try to work that in.
> 
> In the DEP, I didn't see any references to Devuan and Raspbian, is it intended ?

No, just not added at the time. I suppose they could be added as well.

> > Do you want to see regular updates on the mailing list about changes on the `master` branch?
> 
> Yes, but only for the Changelog file, each time this file is updated. It could simply be a diff.

I suppose a script could monitor the repository for changes in the Changelog, extract a diff and send it to the mailing list. It could be run on the same host as the mailing list, too.

**Comment by nqb** [comment]
> I suppose a script could monitor the repository for changes in the Changelog, extract a diff and send it to the mailing list. It could be run on the same host as the mailing list, too.

It is possible to have same behavior with http://log.debops.org ?

**Comment by drybjed** [comment]
Actually, yes! The https://github.com/debops/debops/commits/master/CHANGELOG.rst.atom ATOM feed could be fed to [the `rss2email` script](https://github.com/wking/rss2email), that would probably do the trick. I think that should be pretty easy to set up, I might do it before the next release and send the mails to `debops-users` mailing list. Sounds good?

**Comment by nqb** [comment]
Perfect @drybjed.

**Comment by nqb** [comment]
It could be an opportunity to clean up some old subscriptions on http://log.debops.org and also highlight http://log.debops.org on http://docs.debops.org.

**Comment by drybjed** [comment]
The https://log.debops.org/ site was useful when all of the roles were in separate `git` repositories - the "planet" provided a convenient RSS feed for all repositories. But with everything moving to the single monorepo I don't think there's a need for this anymore.

I'm considering setting up something similar to https://people.kernel.org/ with [WriteFreely](https://writefreely.org/) blog platform, where people that work on/with DebOps could write about their experience. Would that be interesting to have? Of course everything implemented via DebOps.

**Comment by nqb** [comment]
> The https://logs.debops.org/ site was useful when all of the roles were in separate `git` repositories - the "planet" provided a convenient RSS feed for all repositories. But with everything moving to the single monorepo I don't think there's a need for this anymore.

I agree.

> Would that be interesting to have?

Yes. That would be awesome !

**Pull request merged**